Dockerfile

```diff
@@ -10,7 +10,8 @@ ENV NODEJS_VERSION=10 \
     PATH=$HOME/.local/bin/:$HOME/node_modules/.bin/:$HOME/.npm-global/bin/:$PATH \
     LC_ALL=en_US.UTF-8 \
     LANG=en_US.UTF-8 \
-    PIP_NO_CACHE_DIR=off
+    PIP_NO_CACHE_DIR=off \
+    ENABLE_PIPENV=true
```

Comment:
Curious what this change is related to? Will clowder be using Dockerfile.new solely? We do have some UI components in the app currently for viewing the DRF browsable API.

Reply:
So, @bennyturns and I were working together on this at the end, originally @bennyturns was using your setup, which was based on s2i I believe, but @kylape asked me to see if we could put together a new raw Dockerfile, so that we were not based off of s2i anymore. I would imagine this change can be ignored, but would need to ask @bennyturns.

Reply:
I will remove this change.